akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -124,11 +125,12 @@ private[akka] object OneBoundedInterpreter {
 private[akka] class OneBoundedInterpreter(ops: Seq[Stage[_, _]],
                                           onAsyncInput: (AsyncStage[Any, Any, Any], AsyncContext[Any, Any], Any) ⇒ Unit,
                                           materializer: FlowMaterializer,
+                                          attributes: OperationAttributes = OperationAttributes.none,
```

Comment:
We will fix this when implementing the fusing (which is the point at which this solution will stop working anyway).